### PR TITLE
Fix linter warnings

### PR DESCRIPTION
**Pull request by ericgaspar**
- *Fix linter warnings*

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -110,6 +110,13 @@ ln -s "/etc/uwsgi/apps-available/$app.ini" "$final_path/syncserver.ini"
 # configure uwsgi
 ynh_add_uwsgi_service 'domain secret db_user db_pwd db_name'
 
+#=================================================
+# INTEGRATE SERVICE IN YUNOHOST
+#=================================================
+ynh_script_progression --message="Integrating service in YunoHost..."--weight=1
+
+yunohost service add "uwsgi-app@$app" --log "/var/log/uwsgi/$app/$app.log"
```
Comment (Josue-T) [inline scripts/install:118]: Well,

No, It's already managed by the helper here: https://github.com/YunoHost-Apps/ffsync_ynh/blob/testing/scripts/experimental_helper.sh#L109
and there are an issue on the package linter about this: https://github.com/YunoHost/package_linter/issues/89

File: scripts/upgrade [hunk 2]
```diff
@@ -18,7 +18,7 @@ ynh_script_progression --message="Loading installation settings..."
 app=$YNH_APP_INSTANCE_NAME
 
 domain=$(ynh_app_setting_get --app=$app --key=domain)
-path_url=$(ynh_normalize_url_path --path_url $(ynh_app_setting_get --app $app --key path))
+path_url=$(ynh_app_setting_get --app $app --key path)
```
Comment (Josue-T) [inline scripts/upgrade:21]: Well, keep this here because in some really old instance we still could need this.

**Review by Josue-T — CHANGES_REQUESTED** [review]
Thanks for the PR, but I knew these warnings but I wasn't able to fix it cleanly.

**Pull request merged**